labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -70,26 +61,22 @@ object ListManipulationExercise01 {
    * - ... whichever way you like 
    * 
    */
-  def sortList[T <% Ordered[T]](list: List[T]): List[T] = {
-    error("fix me")
-  }
+  def sortList[T <% Ordered[T]](list: List[T]): List[T] = list.sorted
 
   /**
    * Check whether a given element in a list exists, i.e. elementExists(List("a", "b", "c"), "b") = true
    * Again, easy to implement using built-in functionality, but also possible to implement in your own free-style way.
    */
-  def elementExists[T](l: List[T], e: T): Boolean = {
-    error("fix me")
-  }
+  def elementExists[T](l: List[T], e: T): Boolean = if (l.isEmpty) false
+  													else if (l.head == e) true
```

Comment:
Watch out for tabs! I think most people at Mudd are wrong about them always being bad, but they can certainly mess you up if you aren’t careful.